uniauth/views.py

```diff
@@ -662,6 +733,8 @@ def verify_token(request, pk_base64, token):
             LinkedEmail.objects.filter(address=email.address,
                     is_verified=False).delete()
 
+        if api_mode: 
+            return HttpResponseRedirect(get_setting('UNIAUTH_UI_DOMAIN') + '/email/verify/success')
```

Comment:
❓ Is `UNIAUTH_UI_DOMAIN` supposed to be the URL of the uniauth views for the client app? If so, could we replace this with a reverse?